examples/master-detail/app.js

```diff
@@ -27,9 +27,6 @@ const App = React.createClass({
   },
 
   updateContacts() {
-    if (!this.isMounted())
```

Comment:
I think these isMounted checks were no-ops. The example tears down the subscribers on unmount.